AzureCalling/External/Calling/BottomDrawerCellViewModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+struct BottomDrawerCellViewModel {
```

Comment:
This is not really a view model since it contains references to UI classes. Perhaps we can rename it to something like `BottomDrawerItem`.

Reply:
Yukon wanted to rename the class too.  My original thought of naming the class as `ViewModel` because it's [a common usage for tableview cell](https://cocoacasts.com/simplifying-table-views-with-model-view-viewmodel). 
Whatever, I am fine with renaming it